### PR TITLE
Feature/support connection capacity

### PR DESCRIPTION
**Pull request by dancrumb**
Added support for capacity.

Looks like it was mostly there, just needed to add it to the JSON

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,45 +1,45 @@
 {
```
Comment (dancrumb) [inline package.json]: Dumb whitespace changes... sorry :(

File: README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The final semi-colon may be omitted.
 
 Other symbols:
 
-- "Proc-name" represents a process name, optionally followed by the component name (in round brackets), optionally followed by a question mark to indicate that tracing is desired.  The component name can be specified on any occurrence of the process name. 
+- "Proc-name" represents a process name, optionally followed by the component name (in round brackets). The component name can be specified on any occurrence of the process name.
```
Comment (dancrumb) [inline README.md]: It actually doesn't support question marks at the moment.

File: script/fbpscan.js [hunk 3]
```diff
@@ -309,6 +309,10 @@ var fbp = {
 					if (downindex !== '')
 						connx.tgt.index = downindex;
 
+					if(capacity) {
```
Comment (dancrumb) [inline script/fbpscan.js]: This is all it really took!

**Comment by jpaulm** [comment]
I think I have to add this to Travis...?

**Comment by dancrumb** [comment]
can you make me a collaborator on this repo, please?

**Comment by jpaulm** [comment]
Done!

**Pull request merged**